doc/articles/get-started-dotnet-new.md

```diff
@@ -4,7 +4,7 @@ The Uno Platform provides a set of command-line templates to create cross-platfo
 
 To install the templates, type the following:
 ```
-dotnet new install Uno.ProjectTemplates.Dotnet
+dotnet new --install Uno.ProjectTemplates.Dotnet
```

Comment:
@iury-kc Do you have .NET 7 installed?

This command works starting with .NET 7. `dotnet new --install` is deprecated in .NET 7+. See https://github.com/unoplatform/uno/pull/10174

Probably best to reword the documentation and mention both commands

Reply:
Indeed, dotnet new `--install` is deprecated in .NET 7. Maybe we should be adding a note instead to say that in net6 or below it needs to be `-i`.